keyboards/mechlovin/zed65/mono_led/mono_led.c

```diff
@@ -0,0 +1,63 @@
+    {61,            62,            63,                                                         64,                                                         65,            66,                           67,            68,             69},
+}, {
+        //LED Index to Physical Positon
+  {0, 0},     {15,  0},      {30,  0},      {45,  0},      {60,  0},      {75,  0},      {90,  0},      {105, 0},      {119, 0},      {134, 0},      {149, 0},      {164, 0},      {179, 0},      {194, 0},      {209, 0},      {224, 0},
+  {0,16},     {15, 16},      {30, 16},      {45, 16},      {60, 16},      {75, 16},      {90, 16},      {105,16},      {119,16},      {134,16},      {149,16},      {164,16},      {179,16},      {194,16},                     {224,16},
+  {0,32},     {15, 32},      {30, 32},      {45, 32},      {60, 32},      {75, 32},      {90, 32},      {105,32},      {119,32},      {134,32},      {149,32},      {164,32},      {179,32},      {194,32},                     {224,32},
+  {0,48},     {15, 48},      {30, 48},      {45, 48},      {60, 48},      {75, 48},      {90, 48},      {105,48},      {119,48},      {134,48},      {149,48},      {164,48},      {179,48},      {194,48},                     {224,48},
+  {0,64},     {15, 64},      {30, 64},                                                   {90, 64},                                                   {149,64},      {164,64},                     {194,64},      {209,64},      {224,64},
+    }, {
+       1,            1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,
+       1,            1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,                            1,
+       1,            1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,                            1,
+       1,            1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,                            1,
+       1,            1,             1,                                                          1,                                                          1,             1,                            1,             1,             1,
+    } 
+};
+
+
+__attribute__((weak)) void led_matrix_indicators_kb(void) {
+    if (host_keyboard_led_state().caps_lock) {
+        led_matrix_set_value(31, 0xFF);
+    }
+}
```

Comment:
these are switching to boolean this cycle. And at this point, while it may error on master, develop will compile correctly.

```suggestion
bool led_matrix_indicators_kb(void) {
    if (!led_matrix_indicators_user()) {
    	return false;
    }
    if (host_keyboard_led_state().caps_lock) {
        led_matrix_set_value(31, 0xFF);
    }
}
```